### DISCUSSION TITLE
Scripting for server

### DISCUSSION THREAD

**Discussion by martiege** [mkb79/audible-cli#210] [Q&A]
Hello! 

I recently wanted to set up audiobookshelf on my home server, and thus got back into this project! 

Do anyone have any experience with using this library for scripting? I want to write one script to check if I have any new titles and download tgem, and one to check if there are any newly downloaded files, and decrypt them. Then I'll run these monthly using cron. 

These scripts are pretty straightforward, but I was wondering if anyone has any experience doing this on a server? It would be nice, for example, to not set everything up interactively through the quickstart command, so that everything could be done a bit more structured.

**Comment by damajor** [comment]
Check my repo https://damajor.github.io/BALD/
